src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+for (const row of rows) {
+  const rowClone = row.children[1].cloneNode(true);
```

Comment:
JavaScript document has methods to work with tables. [Theory tables](https://javascript.info/dom-navigation#dom-navigation-tables)